pkg/controller/controller_binding.go

```diff
@@ -581,6 +614,16 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 		glog.V(5).Infof("Successfully deleted ServiceInstanceCredential %v/%v of ServiceInstance %v/%v of ServiceClass %v at ServiceBroker %v", binding.Namespace, binding.Name, instance.Namespace, instance.Name, serviceClass.Name, brokerName)
 	}
 
+	if toUpdate.Status.OrphanMitigationInProgress {
+		s := fmt.Sprintf(`Orphan mitigation successful for ServiceInstanceCredential "%v/%v", setting condition to failed`, binding.Namespace, binding.Name)
+		c.setServiceInstanceCredentialCondition(toUpdate,
+			v1alpha1.ServiceInstanceCredentialConditionFailed,
+			v1alpha1.ConditionTrue,
+			"OrphanMitigationSuccessful",
+			s)
+		toUpdate.Status.OrphanMitigationInProgress = false
+		c.updateServiceInstanceCredentialStatus(toUpdate)
```

Comment:
Need to check the return from updateServiceInstanceCredentialStatus so that reconciliation continues if there was a conflict updating the status.